corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -67,6 +67,20 @@ FormplayerFrontend.reqres.setHandler('clearMenu', function () {
     $('#menu-region').html("");
 });
 
+$(document).bind("ajaxStart", function(){
+    var requestElements = $(".formplayer-request");
+    for (var i = 0; i < requestElements.length; i++ ) {
+        requestElements[i].style.pointerEvents = 'none';
+    }
```

Comment:
That doesn't seem to be having the desired affect - maybe doesn't work on not-buttons?